safehttp/doc.go

```diff
@@ -12,5 +12,113 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package safehttp TODO(@kele, @empijei)
+// Package safehttp provides a framework for building secure-by-default HTTP
+// servers. See https://github.com/google/go-safeweb#readme to learn about the
+// goals and features.
+//
+// Safe Responses
+//
+// HTTP responses need to be crafted carefully in order to prevent common web
+// vulnerabilities like Cross-site Scripting (XSS). To help with this, we use
+// Safe Responses.
+//
+// Safe Responses are HTTP responses which has been determined to be safe when
+// received by a modern, popular web browser. For example, we consider HTML
+// responses generated using a template system enforcing contextual autoescaping
```

Comment:
maybe link to https://rawgit.com/mikesamuel/sanitized-jquery-templates/trunk/safetemplate.html or https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/42934.pdf or https://blogtitle.github.io/go-safe-html/ or something like that to give a reference on what the term "contextual autoescaping" means.